commands/commands.go

```diff
@@ -0,0 +1,35 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// commMap maps the name of a command to the function which executes the command
+var commMap = map[string]command{
+	"ping": pingCommand,
+}
+
+// command is a function which executes the given command and arguments on
+// the provided discord session.
+type command func(*discordgo.Session, *discordgo.MessageCreate, []string) error
```

Comment:
maybe wanna have the help part again but we can add that in at a later stage perhaps

Reply:
I never thought of that. I will add it in. 
I'll also add a command for listing all of the commands so that people see what is available to them :)

Reply:
might wanna have a look at message embeds, [heres an example](https://github.com/Strum355/2Bot-Discord-Bot/blob/master/msgUtils.go#L57)